blur.rb

```diff
@@ -0,0 +1,124 @@
+class Image
```

Comment:
Very interesting approach and it isn't bad.  In general though, we're pushing students to solve this problem in one of two slightly different ways.  

# Option 1

This is the easiest.  Take your solution to image blur and just call that method `n` times.

```
def blur(n)
  n.times do
     # old code
  end
end
```

Or way number two, which is similar to your approach, but gets there in a slightly different way (more math based).  The math property on these elements you can say is that the sum of the x/y pairs is less than or equal to the input.

So for:

```
def blur(n)
```

First come up with all the values such that |x| + |y| <= n

So for example:

n = 2:

[0,1], [1, 0], [1, 1], [2, 0], [0, 2]

Then do it for all signs:

[-0,1], [-1, 0], [-1, 1], [-2, 0], [-0, 2]

[0,-1], [1, -0], [1, -1], [2, -0], [0, -2]

[-0,-1], [-1, -0], [-1, -1], [-2, -0], [-0, -2]

Then remove duplicates, and blur all of the pixels x positions away from the starting zone.

Option 2 is basically the similar to the solution you came up with here.